### PR TITLE
After SSCSSI-404 release cleanup

### PR DESCRIPTION
**Pull request by srochani**
### Jira link

See [PROJ-XXXXXX](https://tools.hmcts.net/jira/browse/PROJ-XXXXXX)

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by sanjayparekh-hmcts — CHANGES_REQUESTED** [review]

**Review by SomeshRA — CHANGES_REQUESTED** [review]
I suppose we need to clean up from these charts as well
https://github.com/hmcts/sscs-tribunals-case-api/blob/clean-up-hearing-preview/charts/sscs-tribunals-api/Chart.yaml#L51
https://github.com/hmcts/sscs-tribunals-case-api/blob/clean-up-hearing-preview/charts/sscs-tribunals-api/values.yaml#L179

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -141,7 +136,7 @@ withPipeline(type, product, component) {
     }
     onPR() {
         setPreviewEnvVars();
-
+        echo "isHearingsPr  " + isHearingsPr
```
Comment (sanjayparekh-hmcts) [inline Jenkinsfile_CNP:139]: need this?

**Review by sanjayparekh-hmcts — APPROVED** [review]

File: Jenkinsfile_CNP (hunk 1 above)
Comment (srochani) [reply Jenkinsfile_CNP:139]: needed for debug :-)

File: src/main/java/uk/gov/hmcts/reform/sscs/jms/listener/HmcHearingsEventTopicListener.java [hunk 2]
```diff
@@ -54,14 +54,10 @@ public HmcHearingsEventTopicListener(@Value("${sscs.serviceCode}") String sscsSe
     )
     public void onMessage(JmsBytesMessage message) throws JMSException, HmcEventProcessingException {
 
-        log.info("isByPassHearingServiceEnabled && isDeploymentFilterEnabled ------------------------> {}, {}", isByPassHearingServiceEnabled,
```
[the remark this answers is not in the document]
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/sscs/jms/listener/HmcHearingsEventTopicListener.java]: updated to keep this log.info

File: src/main/java/uk/gov/hmcts/reform/sscs/jms/listener/TribunalsHearingsEventQueueListener.java [hunk 3]
```diff
@@ -40,13 +40,10 @@ public class TribunalsHearingsEventQueueListener {
     )
     public void handleIncomingMessage(HearingRequest message) throws TribunalsEventProcessingException, GetCaseException, UpdateCaseException {
 
-        log.info("isByPassHearingServiceEnabled ------------------------> {}", isByPassHearingServiceEnabled);
```
[the remark this answers is not in the document]
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/sscs/jms/listener/TribunalsHearingsEventQueueListener.java]: updated to keep this log.info

**Review by gokul-sol — APPROVED** [review]

**Pull request merged**